### ISSUE TITLE
New Analysis Section Idea: Patterns and Pitfalls

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/refinebio-examples#374]
### What are the goals of this new example analysis?

As we identify common patterns that people encounter in refine.bio, we should think about ways we can address these in examples. Here I am thinking of things that are not a particular analysis, but that would be common to multiple potential analyses. In a way, I am thinking about some of the things that are in the FAQ, but might not be well addressed in that context. Think of it as the online supplemental materials for the FAQ.

I am calling it Patterns and Pitfalls, because I started thinking about it as common pitfalls, but that was too negative, and alliteration always amazes.

Some examples of things I am thinking about (I am sure there are more!):

* Replicates in a data set (this was the original inspiration)
  * Identifying technical replicates (may include querying SRA?)
  * collapsing for RNAseq (for technical replicates)
  * specifying more complex models for DE?

* Identifying and dropping outlier samples
  * Look at a PCA plot, perhaps?

* I might move identifier mapping to this section?

* Other common challenges that Deepa might have identified?

This is not a "before live" problem.

**Comment by jaclyn-taroni** [comment]
To clarify, when you say FAQ do you mean #375 or https://github.com/AlexsLemonade/refinebio-docs/blob/master/docs/faq.md? I assume the latter, which is more about potential pitfalls than what's in #375.

**Comment by jashapiro** [comment]
Yeah, I was thinking about the [refine.bio FAQ](http://docs.refine.bio/en/latest/faq.html), not the potential examples FAQ that I filed after this one....

**Comment by jaclyn-taroni** [comment]
This may be better conceptualized as something that is standalone rather than part of our existing structure (intro, microarray, RNA-seq, advanced) that we "funnel" users to. (A bit of thought required about _how_ to do that of course.) I'm think something like a Bioconductor vignette might be the right format. Here are a few examples to help illustrate what I mean:

* [Single-cell analysis toolkit for expression in R](https://bioconductor.org/packages/release/bioc/vignettes/scater/inst/doc/overview.html)
* [Using scran to analyze single-cell RNA-seq data](https://bioconductor.org/packages/release/bioc/vignettes/scran/inst/doc/scran.html)
* [Beginner's guide to using the DESeq2 package](https://bioc.ism.ac.jp/packages/2.14/bioc/vignettes/DESeq2/inst/doc/beginner.pdf)

Another way of putting it might be "one long file" with a TOC that covers all the patterns and pitfalls that is largely unconnected to what else we have (e.g., doesn't share a navbar).

**Comment by cansavvy** [comment]
Another issue that might be best served in this kind of format/vignette type deal is the metadata cleaning tips: https://github.com/AlexsLemonade/refinebio-examples/issues/210
